base62.go

```diff
@@ -25,28 +25,54 @@ func base2base(in []byte, inBase int, outBase int) []byte {
 		}
 
 		// this is really a prepend or remainder to result
-		res = append([]byte{byte(remainder)}, res...)
+		dst = append(dst, byte(remainder))
 		bs = quotient
 	}
 
-	return res
+	reverse(dst)
```

Comment:
None that I know of... but I think it's worth it, in the case of a ksuid it'll do 13 loop iterations, which isn't too bad 😛